src/backend/distributed/connection/remote_commands.c

```diff
@@ -492,7 +492,7 @@ SendRemoteCommandParams(MultiConnection *connection, const char *command,
 	Assert(PQisnonblocking(pgConn));
 
 	int rc = PQsendQueryParams(pgConn, command, parameterCount, parameterTypes,
-							   parameterValues, NULL, NULL, 0);
+							   parameterValues, NULL, NULL, binaryResults ? 1 : 0);
```

Comment:
I think here using something more readable instead of `1` and `0` which are mappings to binary format and text format could be better. 

For example:

```c
#define BINARY_FORMAT 1
#define TEXT_FORMAT 0
```